src/components/chip/chip.tsx

```diff
@@ -98,14 +131,12 @@ export class Chip
     /* wired up by t9n util */
   }
 
-  // --------------------------------------------------------------------------
+  //--------------------------------------------------------------------------
   //
-  //  Private State/Properties
+  //  Private Properties / State
```

Comment:
Nitpick: Just noticed we're pretty inconsistent with this. Most files use `Private State/Props`.